src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {

```

Comment:
You wrote the logic here that you should have in the FigureSuplier methods. I'll write it there, and use the methods here